longfi-us915/longfi-us915.ino

```diff
@@ -35,6 +35,15 @@
 #include <arduino_lmic.h>
 #include <hal/hal.h>
 #include <SPI.h>
+#include <Adafruit_GPS.h>
+
+#define GPSSerial Serial1
+
+#define CFG_sx1276_radio 1
```

Comment:
is this necessary? we probably don't want to do this

Reply:
The GPSSerial I think it is, the CFG_sx1276_radio can be removed